.packer/ami-version.txt

```diff
@@ -1 +1 @@
-1.1.0
+2.0.1
```

Comment:
This is a major/next generation change to the AMI so I've reflected that here

It changes:

1. To a much newer specification of C/C++
2. Completely changes the underlying base ami and the supported instance type architectures
3. Brings us way up-to-date with CUDA drivers/implementations